src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you already use these properties on your class(`stars__star`), do you need to duplicate it?

Reply:
In both cases we need the picture to work properly, so of course we need these lines and it is not an unnecessary duplicate

Reply:
yes, but we already have that behavior above on `stars__star` class)
u can do it without duplicated. Leave it for the end, edit all the comments, and try